### PR TITLE
refactor: edit readme

### PR DESCRIPTION
**Pull request by xenowits**
1. Remove README section for building from source.
2. Change default location of `cluster-lock.json`.

category: docs
ticket: none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,16 +25,18 @@ which contains a docker compose setup for running a charon cluster on your local
...
+# Install go 1.18 or later (https://go.dev/doc/install)
+# If on mac, you can also install using homebrew
 brew install go
 
 # Build the charon binary
 go build -o charon
 
-# Use charon's create-cluster command to generate a local simnet cluster.
+# Use charon's create cluster command to generate a local charon cluster
+./charon create cluster
```
Comment (corverroos) [inline README.md:36]: not sure this is very helpful, what is the dev going to do with a cluster_lock file? Suggesting removing this section and just pointing them to charon-docker-compose

Comment (xenowits) [reply README.md:36]: so, remove the whole "build directly from source" section?

File: cmd/run.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func newRunCmd(runFunc func(context.Context, app.Config) error) *cobra.Command {
 }
 
 func bindRunFlags(flags *pflag.FlagSet, config *app.Config) {
-	flags.StringVar(&config.LockFile, "lock-file", ".charon/cluster-lock.json", "The path to the cluster lock file defining distributed validator cluster")
+	flags.StringVar(&config.LockFile, "lock-file", ".charon/cluster/cluster-lock.json", "The path to the cluster lock file defining distributed validator cluster")
```
Comment (corverroos) [inline cmd/run.go:57]: not sure about this. What is the reasoning behind the change?

Comment (xenowits) [reply cmd/run.go]: sorry, unwanted change (in my head, i was thinking of charon create cluster which puts cluster-lock.json inside .charon/cluster/cluster-lock.json)

**Pull request merged**